anomaliesDetection/products.go

```diff
@@ -260,8 +260,5 @@ func productGetAnomaliesData(ctx context.Context, params AnomalyEsQueryParams) (
 		totalAnalyzedCosts = append(totalAnalyzedCosts, aCosts...)
 	}
 	totalAnalyzedCosts = productClearDisturbances(totalAnalyzedCosts, totalCostsByDay, highestSpendersByDay)
-	if err != nil {
```

Comment:
Why has this part of the code been removed?

Reply:
It could stay, but it's not doing anything: `err` will always be `nil` when it is reached. I suppose this might be more appropriate for another PR, should you consider it very important to put it somewhere else like that.

Reply:
You right 👍